api/src/main/java/com/codeforcommunity/rest/subrouter/CommonRouter.java

```diff
@@ -24,8 +25,8 @@ public CommonRouter(JWTAuthorizer jwtAuthorizer) {
   public Router initializeRouter(Vertx vertx) {
     Router router = Router.router(vertx);
 
+    router.route().handler(LoggerHandler.create()); // Adds request logging
```

Comment:
Where does it log requests? This is not the Slack notification thing right?

Reply:
No this just prints to console whenever an incoming request is made